cluster-autoscaler/cloudprovider/clusterapi/clusterapi_utils.go

```diff
@@ -29,6 +29,10 @@ import (
 )
 
 const (
+	capiNodeGroupMinSizeAnnotationKey = "cluster.x-k8s.io/cluster-api-autoscaler-node-group-min-size"
```

Comment:
This is needed for the fallback nature of this commit. The addition here is so that we can check for the CAPI group if the MAPI group doesn't exist. Using `nodeGroupMinSizeAnnotationKey` will only return the MAPI version in a MAPI based cluster.

If we are dropping these, then we may as well just drop this commit entirely (which, looking at the reason it was originally added, may actually be suitable)